app/Http/Controllers/SportsController.php

```diff
@@ -16,17 +17,48 @@ public function create()
         return view('sports.create', compact('sports', 'countries'));
     }
 
-    public function store(Request $request)
+    /**
+     * Sports store method
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function store(Request $request): RedirectResponse
     {
-        // Add your code here
+        $values = $request->except('_token');
+
+        foreach ($values as $key => $countryId) {
+            if ($countryId) {
+                $keyExplode = explode('-', $key);
+
+                $place = $keyExplode[0];
+                $sport = Sport::find($keyExplode[1]);
```

Comment:
@Evolyuta this will fire a database query for each loop iteration, it's better to get sports into collection upfront, and then do find from collection